### ISSUE TITLE
support tox in elpy-test

### ISSUE THREAD

**Issue by anarcat** [jorgenschaefer/elpy#1840]
It would be great if one of the options in the `elpy-test-runner` customization would be `tox`. Alternatively, why don't we just allow users to set an arbitrary string there instead?

**Comment by galaunay** [comment]
Not all test packages have the same syntax to select a given test to run (e.g. `unittest` will run with `python -m unittest tests.test_points.TestPoints.test_copy` and `pytest` with `py.test tests/test_points.py::TestPoints::test_copy`).
That's why we have dedicated functions to build test command lines.

Tox is a bit different as it defer the test to another package.
Depending on which test package you are using in tox (let's `py.test` for the sake of the example), you can overwrite the corresponding test runner command line:
```
(setq elpy-test-pytest-runner-command '("tox"))  ;; if you use py.test
```

**Comment by anarcat** [comment]
yeah, it's pretty much what i did, but it's really counter-intuitive. what i meant was to allow an arbitrary string as an option to the customization... just to make this easier to configure/discover for people who might be less familiar with the package and/or elisp.

**Comment by galaunay** [comment]
It is possible to allow `elpy-test-runner` to be set to an arbitrary string.
But in this case, Elpy will not be able to provide the "test at point" feature (as it will not be aware of the syntax to specify the test to run).

**Comment by anarcat** [comment]
> It is possible to allow elpy-test-runner to be set to an arbitrary string.

Not through `customize`, as far as I can tell.
